### ISSUE TITLE
508-defect-1: Table isn't responsive

### ISSUE THREAD

**Issue by jaredcunha** [department-of-veterans-affairs/va.gov-team#50581]
Single accessibility issue copied from https://github.com/department-of-veterans-affairs/va.gov-team/issues/45650

### Blocked by Apple bug
- [AX: Tables with display properties are skipped, report wrong col/row counts](https://bugs.webkit.org/show_bug.cgi?id=257458)

### Table isn't responsive (Critical)
The first table is not responsive and doesn't follow VA design system guidance
<img width="740" alt="responsivetables" src="https://user-images.githubusercontent.com/14154792/184412722-c1ae3049-6bb7-41fc-8bcb-8e289d2cca41.png">. [Switching to the DST responsive table will create a separate critical issue as VoiceOver does not announce content.](https://github.com/department-of-veterans-affairs/va.gov-team/issues/50934)

### Acceptance criteria
- [ ] Table is responsive

## Reproduction steps

1. [Log in as user 299 and get to step 2](https://staging.va.gov/records/download-va-letters/letters/letter-list)
2. Open the benefit summary accordion

## Type of issue

- [x] Screenreader
- [x] Keyboard
- [ ] Focus
- [x] Headings
- [ ] Color
- [x] Zoom
- [x] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by sortizsh** [comment]
There is no short term solution for this, it needs research and design.

**Comment by sortizsh** [comment]
band aid: instead of a table it could be repeating blocks of bold text in a column with value to right. this would get us around tables being unresponsive. @sortizsh open up a bandaid ticket and link to this one and larger epic.
